frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -58,30 +59,27 @@ const FarmInfoComponent: React.FC<InputModuleProps> = ({ farmDetails, updateFarm
     >
       <Form>
         <StyledFarmInfo>
-          <label htmlFor="FarmName">
-            <p>Name</p>
-          </label>
-          <Field
-            id="FarmName"
-            name="FarmName"
-            type="text"
-          />
-          <ErrorMessage name="FarmName" />
+          <div id="inputContainer">
+            <CustomField
+              label="Year"
+              id="Year"
+              name="Year"
+              type="number"
+              width="20%"
+            />
 
-          <label htmlFor="Year">
-            <p>Year</p>
-          </label>
-          <Field
-            id="Year"
-            name="Year"
-            type="number"
-          />
-          <ErrorMessage name="Year" />
+            <CustomField
+              label="Farm Name"
+              id="FarmName"
+              name="FarmName"
+              type="text"
+            />
+          </div>
 
           <label htmlFor="FarmRegion">
             <p>Region</p>
```

Comment:
Inconsistencies with the other CustomField Labels, This is not bolded as other labels.

Reply:
What do you mean? For the Select label?
The custom Select component is my next ticket, not a part of this one :wink:

Reply:
Thanks for correcting me in that! You're right

Reply:
It's confusing mate, no worries